functions/report-psychometrician/service/date.service.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
```

Comment:
why is this under this function service catalog ?
Sounds like this could be reused in the future by other functions as well? 🤔

Reply:
It's been pared down to just what this function requires - a vertical slice.  If other functions needed a date service there would be duplicate data.service.js files,  and there aren't.